Orange/evaluation/testing.py

```diff
@@ -366,21 +365,21 @@ def __init__(self, data, learners, store_data=False, store_models=False,
         self.call_callback(1)
 
 
-class Bootstrap(Results):
```

Comment:
Would you consider wrapping the new interface in the old, deprecated? Something like
```py
@deprecated
class Bootstrap(Results):
    def __init__(...):
        ... ShuffleSplit(...) ...
```